src/components/templates/Search/index.tsx

```diff
@@ -7,6 +7,43 @@ import queryString from 'query-string'
 import { getResults } from './utils'
 import Loader from '../../atoms/Loader'
 import { useOcean } from '@oceanprotocol/react'
+import { DragDropContext, Droppable, Draggable } from 'react-beautiful-dnd'
+const defaultSearchItems = () =>
+  ['created', 'price.ocean', 'price.value'].map((e) => ({
+    id: e,
+    direction: -1
+  }))
+
+// a little function to help us with reordering the result
```

Comment:
Sorting the results on client could lead to many problems, but more importantly we need to reorder across all search result pages, not just for the current page. So every change in sorting simply needs to fire a modified search query to Aquarius